addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -61,6 +61,10 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
     }
 }
 
+.o_ws_nofooter {
+    padding-bottom: 32px !important;
+}
```

Comment:
It should probably be on the `<main>` instead of the \#wrap but even there that will create problems in boxed layouts where those elements are colored: example, on the cart page there is an oe_structure at the end, allowing you to fill the area with snippets... but with this fix you fill have a 32px tall band of color you can't remove :grimacing: 

But I can't think of another good solution in 5min... so if you are ok with that problem for now, I let you judge